components/layout/DefaultLayout.tsx

```diff
@@ -61,10 +61,10 @@ export default function DefaultLayout(props: {
   }
 
   function responseMessage(token: CredentialResponse): void {
-    console.log(token);
+    // console.log(token);
     if (token.credential !== undefined) {
       const jwt: UserInfo = jwtDecode(token.credential);
-      console.log(jwt);
+      // console.log(jwt);
```

Comment:
Can we delete the log statements instead of commenting them out?

Reply:
Sure